### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by cmseasy** [Skons/hassio-addons#47]
Trying to install and have this error:

`Failed to to call /addons/f5a8083e_dutch_gas_prices/install - The command '/bin/bash -o pipefail -c apk add --no-cache --virtual .build-dependencies tesseract-ocr tesseract-ocr-dev jpeg-dev libjpeg zlib-dev gcc libc-dev' returned a non-zero code: 88`

Install device: Intell NUC with Homeassistant (with supervisor)  perfect working in a Proxmox VM. 
HA migrated from a Odroid  N2+ with a working dgp. It breaks in the NUC and will not install.

**Comment by Skons** [comment]
Fresh installation?

**Comment by cmseasy** [comment]
Fresh HA installation on the NUC and uploaded HA backup from the Odroid device.
All is working ok, only the dutch_gas_prices addon breaks. I removed the addon and reinstalling gives the error.

**Comment by mupsje** [comment]
yes, also same here again....

<img width="428" alt="image" src="https://github.com/Skons/hassio-addons/assets/46684538/f708e58b-a63c-4b67-b959-6b231028f7b3">

greets

```
The command '/bin/bash -o pipefail -c apk add --no-cache --virtual .build-dependencies tesseract-ocr tesseract-ocr-dev jpeg-dev libjpeg zlib-dev gcc libc-dev' returned a non-zero code: 88
```

**Comment by Skons** [comment]
Finally i got some time to at least reproduce the issue. I will try to fix this asap
```
tep 7/27 : RUN     apk add --no-cache --virtual .build-dependencies         tesseract-ocr         tesseract-ocr-dev         jpeg-dev         libjpeg         zlib-dev         gcc         libc-dev
 ---> Running in a7b41a97ea24
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/community/x86_64/APKINDEX.tar.gz
ERROR: unable to select packages:

  musl-1.2.3-r4:
    breaks: musl-dev-1.2.3-r5[musl=1.2.3-r5]
    satisfies: world[musl=1.2.3-r4]
               libbsd-0.11.7-r0[musl]
               libbsd-0.11.7-r0[so:libc.musl-x86_64.so.1]
               apk-tools-2.12.10-r1[musl>=1.2]
               apk-tools-2.12.10-r1[so:libc.musl-x86_64.so.1]
               pkgconf-1.9.4-r0[so:libc.musl-x86_64.so.1]
               libblkid-2.38.1-r1[so:libc.musl-x86_64.so.1]
               dash-0.5.11.5-r2[so:libc.musl-x86_64.so.1]
               gcc-12.2.1_git20220924-r4[so:libc.musl-x86_64.so.1]
               graphite2-1.3.14-r2[so:libc.musl-x86_64.so.1]
               freetype-2.12.1-r0[so:libc.musl-x86_64.so.1]
               leptonica-1.82.0-r1[so:libc.musl-x86_64.so.1]
               libffi-3.4.4-r0[so:libc.musl-x86_64.so.1]
               libxext-1.3.5-r0[so:libc.musl-x86_64.so.1]
               libjpeg-turbo-2.1.4-r0[so:libc.musl-x86_64.so.1]
               libpng-1.6.38-r0[so:libc.musl-x86_64.so.1]
               bash-5.2.15-r0[so:libc.musl-x86_64.so.1]
               
ca-certificates-20220614-r4[so:libc.musl-x86_64.so.1]
               libxau-1.0.10-r0[so:libc.musl-x86_64.so.1]
               binutils-2.39-r2[so:libc.musl-x86_64.so.1]
               busybox-1.35.0-r29[so:libc.musl-x86_64.so.1]
               giflib-5.2.1-r2[so:libc.musl-x86_64.so.1]
               tesseract-ocr-5.2.0-r1[so:libc.musl-x86_64.so.1]
               sqlite-libs-3.40.1-r0[so:libc.musl-x86_64.so.1]
               libmd-1.0.4-r0[so:libc.musl-x86_64.so.1]
               libuuid-2.38.1-r1[so:libc.musl-x86_64.so.1]
               pango-1.50.13-r0[so:libc.musl-x86_64.so.1]
               keyutils-libs-1.6.3-r1[so:libc.musl-x86_64.so.1]
               curl-8.0.1-r0[so:libc.musl-x86_64.so.1]
               cairo-1.17.6-r3[so:libc.musl-x86_64.so.1]
               krb5-libs-1.20.1-r0[so:libc.musl-x86_64.so.1]
               libx11-1.8.4-r0[so:libc.musl-x86_64.so.1]
               libxcb-1.15-r0[so:libc.musl-x86_64.so.1]
               ncurses-libs-6.3_p20221119-r0[so:libc.musl-x86_64.so.1]
               musl-utils-1.2.3-r4[so:libc.musl-x86_64.so.1]
               nghttp2-libs-1.51.0-r0[so:libc.musl-x86_64.so.1]
               isl25-0.25-r1[so:libc.musl-x86_64.so.1]
               libstdc++-12.2.1_git20220924-r4[so:libc.musl-x86_64.so.1]
               libbz2-1.0.8-r4[so:libc.musl-x86_64.so.1]
               libxrender-0.9.11-r0[so:libc.musl-x86_64.so.1]
               brotli-libs-1.0.9-r9[so:libc.musl-x86_64.so.1]
               libcom_err-1.46.6-r0[so:libc.musl-x86_64.so.1]
               libwebp-1.2.4-r2[so:libc.musl-x86_64.so.1]
               harfbuzz-5.3.1-r1[so:libc.musl-x86_64.so.1]
               libcurl-8.0.1-r0[so:libc.musl-x86_64.so.1]
               mpc1-1.2.1-r1[so:libc.musl-x86_64.so.1]
               fontconfig-2.14.1-r0[so:libc.musl-x86_64.so.1]
               gdbm-1.23-r0[so:libc.musl-x86_64.so.1]
               icu-libs-72.1-r1[so:libc.musl-x86_64.so.1]
               oniguruma-6.9.8-r0[so:libc.musl-x86_64.so.1]
               mpfr4-4.1.0-r0[so:libc.musl-x86_64.so.1]
               libcrypto3-3.0.8-r3[so:libc.musl-x86_64.so.1]
               
zlib-1.2.13-r0[so:libc.musl-x86_64.so.1]
               libmount-2.38.1-r1[so:libc.musl-x86_64.so.1]
               glib-2.74.7-r0[so:libc.musl-x86_64.so.1]
               libxft-2.3.7-r0[so:libc.musl-x86_64.so.1]
               libgomp-12.2.1_git20220924-r4[so:libc.musl-x86_64.so.1]
               libintl-0.21.1-r1[so:libc.musl-x86_64.so.1]
               readline-8.2.0-r0[so:libc.musl-x86_64.so.1]
               libxdmcp-1.1.4-r0[so:libc.musl-x86_64.so.1]
               zstd-libs-1.5.5-r0[so:libc.musl-x86_64.so.1]
               libgcc-12.2.1_git20220924-r4[so:libc.musl-x86_64.so.1]
               libnsl-2.0.0-r0[so:libc.musl-x86_64.so.1]
               fribidi-1.0.12-r0[so:libc.musl-x86_64.so.1]
               libssl3-3.0.8-r3[so:libc.musl-x86_64.so.1]
               libexpat-2.5.0-r0[so:libc.musl-x86_64.so.1]
               libatomic-12.2.1_git20220924-r4[so:libc.musl-x86_64.so.1]
               xz-libs-5.2.9-r0[so:libc.musl-x86_64.so.1]
               pcre2-10.42-r0[so:libc.musl-x86_64.so.1]
               
libverto-0.3.2-r1[so:libc.musl-x86_64.so.1]
               scanelf-1.3.5-r1[so:libc.musl-x86_64.so.1]
               gmp-6.2.1-r2[so:libc.musl-x86_64.so.1]
               libtirpc-1.3.3-r0[so:libc.musl-x86_64.so.1]
               jq-1.6-r2[so:libc.musl-x86_64.so.1]
               tiff-4.4.0-r3[so:libc.musl-x86_64.so.1]
               pixman-0.42.2-r0[so:libc.musl-x86_64.so.1]
               .python-rundeps-20230417.072338[so:libc.musl-x86_64.so.1]
  .build-dependencies-20230531.221208:
    masked in: cache
    satisfies: world[.build-dependencies=20230531.221208]
Removing intermediate container a7b41a97ea24
```

**Comment by Skons** [comment]
Issue is fixed with 2023.6.1.1

**Comment by cmseasy** [comment]
Skons,

Dgp is working again, good job, thanks.


Martin
